src/client/driver/index.js

```diff
@@ -4,7 +4,7 @@ import testCafeUI from './deps/testcafe-ui';
 
 import MESSAGE from '../../test-run/client-messages';
 import COMMAND_TYPE from '../../test-run/commands/type';
-import { UncaughtErrorOnPage } from '../../errors/test-run';
+import { UncaughtErrorOnPage, HybridFnExecutionInterrupted } from '../../errors/test-run';
```

Comment:
It should be `ClientCodeExecutionIterruptionError`. We use generic term `client code` to share error beetween hybrids and `t.eval`